packages/microsoft-reactnative-sampleapps/windows/SampleAppCS/MainPage.xaml

```diff
@@ -1,3 +1,5 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
>All rights reserved. [](start = 42, length = 20)

From:
[https://docs.opensource.microsoft.com/changelog.html?q=all%20rights%20reserved](https://docs.opensource.microsoft.com/changelog.html?q=all%20rights%20reserved)

CELA open source policy
The term All Rights Reserved is no longer recommended to be in MIT license copyright headers.

Reply:
It is addressed by PR #5083

---
In reply to: [433393589](https://github.com/microsoft/react-native-windows/pull/5071#discussion_r433393589) [](ancestors = 433393589)